afs/afs-storage-api/src/main/java/com/powsybl/afs/storage/NodeInfo.java

```diff
@@ -54,6 +54,18 @@ public String getName() {
         return name;
     }
 
+    public void setName(String name) {
+        this.name = checkName(name);
+    }
+
+    private static String checkName(String name) {
+        Objects.requireNonNull(name);
+        if (Objects.equals("", name)) {
```

Comment:
name.isEmpty()

Reply:
Do i neccessary have to implement the rename Method in DefaultListenableAppStorage class (so to create a NodeRenameUpdated class which extends NodeEvent)?
thanks

Reply:
Good point, yes it has to be implemented